velox/exec/SortBuffer.cpp

```diff
@@ -287,9 +287,11 @@ void SortBuffer::spillOutput() {
 void SortBuffer::prepareOutput(uint32_t maxOutputRows) {
   VELOX_CHECK_GT(maxOutputRows, 0);
```

Comment:
maxOutputRows may be used as batchSize, we need to convert it to vector_size_t as check, to make sure it's less than 0x7fffffff

Reply:
Yes, I will create a separate PR to solve it, it involves much refactor